packages/design-system-react-native/src/components/AvatarAccount/AvatarAccount.stories.tsx

```diff
@@ -1,10 +1,10 @@
 import type { Meta, StoryObj } from '@storybook/react-native';
 import { View } from 'react-native';
 
-import { AvatarAccountSize, AvatarAccountVariant } from '../../types';
 import { AvatarAccount } from './AvatarAccount';
 import { SAMPLE_AVATARACCOUNT_ADDRESSES } from './AvatarAccount.constants';
 import type { AvatarAccountProps } from './AvatarAccount.types';
+import { AvatarAccountSize, AvatarAccountVariant } from '../../types';
```

Comment:
@georgewrmarshall previously I've maintained it so that the import order goes from outermost inward, then alphabetized. Lmk if you still want to maintain this pattern. If so we can sync regarding updating the eslint rules

Reply:
Great catch! I did some digging and it looks like the current eslint base config has a bug. I've updated the eslint config to adhere to "outermost inward" pattern and created an issue here https://github.com/MetaMask/eslint-config/issues/403